sycl/source/device.cpp

```diff
@@ -28,6 +26,7 @@ void force_type(info::device_type &t, const info::device_type &ft) {
                                         PI_ERROR_INVALID_OPERATION);
   }
 }
+
```

Comment:
```suggestion
```

Reply:
It somehow came back.
I removed this empty line before.
I will make sure this empty line is removed in the next PR.